setup.cfg

```diff
@@ -74,6 +74,7 @@ all =
     scikit-image
     PyAVM
     astrodendro
+    h5py>=2.10
```

Comment:
Why do you list h5py multiple times? Is the spec in `install_requires` not enough?

Reply:
This would still install it in 3.11 with `glueviz[all]` (I hope); the `platform_system=="Linux"` in `test` below is added as an `OR` so we are still running the full tests with `linux-py311`; the second `python_version<'3.11'` there is redundant, admittedly.